youtube_dl/extractor/generic.py

```diff
@@ -2438,6 +2453,12 @@ def _playlist_from_matches(matches, getter=None, ie=None):
             return _playlist_from_matches(
                 openload_urls, ie=OpenloadIE.ie_key())
 
+        # Look for Visir embeds
+        visir_urls = VisirMediaIE._extract_urls(webpage)
```

Comment:
Generic embed is an embed that may be embedded anywhere. If you can't provide any example URL (apart from visir.is site itself) of embed with such code then it's not a generic embed.